v4/source/system.yaml

```diff
@@ -1004,3 +1004,77 @@
                 format: binary
         "404":
           $ref: "#/components/responses/NotFound"
+  /warn_metrics/status:
+    get:
+      tags:
+        - system
+      summary: Get the warn metrics status (enabled or disabled)
+      description: >
+        Get the status of a set of metrics (enabled or disabled) from the Systems table.
+
+        The returned JSON contains the metrics that we need to warn the admin on with regard to their status (we return the ones whose status is "true", which means that they are in a "warnable" state - e.g. a threshold has been crossed or some other condition has been fulfilled).
+
+        __Minimum server version__: 5.25
+
+        ##### Permissions
+
+        Must have `manage_system` permission.
+      parameters:
+      responses:
+        "200":
+          description: Warn metrics retrieval was successful.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StatusOK"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+    /warn_metrics/ack/{warn_metric_id:[A-Za-z-_]+}":
+    post:
+      tags:
+        - system
+      summary: Acknowledge a warning of a metric status
+      description: |
+        Acknowldge a warning for the warn_metric_id metric crossing a threshold (or some siliar condition being fulfilled) - attempts to send an ack email to acknowledge@mattermost.com and set the "ack" status for the warn metric with "warn_metric_id" id.
+
+        ##### Permissions
+        Must have `manage_system` permission.
+
+        __Minimum server version__: 5.25
```

Comment:
This is in an inconsistent order, it should be before permissions to be equal to the rest of the API documentation.